completion/__init__.py

```diff
@@ -2,6 +2,6 @@
 Completion App
 """
 
-__version__ = '4.4.0'
+import django
```

Comment:
Is it a leftover from the previous commit? Please only do force-pushes (other than branch rebases) after the end of the review. Doing it earlier makes changes harder to track.